CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ string (TOUPPER ${CMAKE_BUILD_TYPE} CMAKE_BUILD_TYPE_UC)
 list(REVERSE CMAKE_FIND_LIBRARY_SUFFIXES)
 
 option (ENABLE_FUZZING "Fuzzy testing using libfuzzer" OFF)
+option (ENABLE_BUZZHOUSE "Enable BuzzHouse fuzzer on the client" OFF)
```

Comment:
This is a great addition to allow building the client without `BuzzHouse`. However, to make sure our CI builds it, either we set it to ON by default, or we modify the build options to enable `BuzzHouse`

Reply:
Ok, I suggested to enable it here for now to make it easy, but my point was that I'd like to see the CI building the same thing we're both building on our dev machines to ensure it works.

So, feel free to modify the CI build instead to make sure this builds correctly.